utils/format.go

```diff
@@ -231,6 +231,9 @@ func formatCurrencyString(valIf interface{}, valueCurrency, targetCurrency strin
 		// add trailing zeros to always have the same amount of digits after the comma
 		dotIndex := strings.Index(valStr, ".")
 		if dotIndex >= 0 {
+			if !strings.HasSuffix(amountStr, ".") {
+				amountStr += "."
+			}
```

Comment:
**issue:** This doesn't make sense to me. What are you trying to do here?
If you now e.g. call this method with:

- valIf: 100.12345
- digitsAfterComma: 10

you would get "100.12345.0000"

or with
- digitsAfterComma: 2

a "100.12." (with a correct tooltip)

Didn't it just work beforehand?

Reply:
Not quite, seems like I came across an edge case which isn't handled correctly. Call the method with
- valIf: 100.00001234
- digitsAfterComma: 4 

result:
- 10000000

When the number has higher precision than `digitsAfterComma`, `Truncate` cuts away *all* decimal places. We're still adding the `missingZeros` though, so we'll massively inflate the result :sweat_smile: 

My fix was also wrong of course, thanks for pointing it out; I've adjusted it.

Reply:
I see, good find.
Seems to work for every case now.